CRM/Upgrade/Incremental/php/FiveFortyNine.php

```diff
@@ -60,6 +60,24 @@ public function upgrade_5_49_beta1($rev): void {
     }
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_49_2($rev) {
+    $this->addtask('Revert civicrm_action_schedule.limit_to to be NULL', 'changeBooleanColumnLimitTo');
+  }
+
+  /**
+   * Revert boolean default civicrm_action_schedule.limit_to to be NULL
+   */
+  public static function changeBooleanColumnLimitTo() {
+    CRM_Core_DAO::executeQuery("ALTER TABLE `civicrm_action_schedule` CHANGE `limit_to` `civicrm_action_schedule` tinyint NULL COMMENT 'Is this the recipient criteria limited to OR in addition to?'", [], TRUE, NULL, FALSE, FALSE);
```

Comment:
You are actually renaming the field from limit_to to civicrm_action_schedule here